### PR TITLE
[ADD] l10n_hu: update CoA

### PR DESCRIPTION
**Pull request by rugo-odoo**
Updated chart of accounts
Add account groups
Updated taxes and tax report

Balance sheet and profit loss are in enterprise

Task: https://www.odoo.com/web#id=2678497&cids=1&menu_id=4720&action=333&active_id=967&model=project.task&view_type=form

Current behavior before PR: Hungarian chart of accounts, taxes, tax report are outdated and there is no balance sheet and profit and loss report.

Desired behavior after PR is merged: Hungarian chart of accounts, taxes and tax report are updated based on current laws and balance sheet and profit and loss report are available.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by william-andre** [comment]
@jbw-odoo I don't know if I should remove you as a reviewer or if you want to review it?
If you don't want to (or if this one doesn't interest you), you can remove (and unsubscribe) yourself

**Review by jbw-odoo — APPROVED** [review]

File: addons/l10n_hu/data/account.fiscal.position.tax.template.csv [hunk 1]
```diff
@@ -3,21 +3,19 @@
...
+"fiscal_position_hu_eu_F27","F27","FEUT","fiscal_position_hu_eu"
+"fiscal_position_hu_eu_F18","F18","FEUT","fiscal_position_hu_eu"
+"fiscal_position_hu_eu_F5","F5","FEUT","fiscal_position_hu_eu"
+"fiscal_position_hu_eu_V27","V27","VEU27T","fiscal_position_hu_eu"
+"fiscal_position_hu_eu_V27TE","V27TE","VEU27TE","fiscal_position_hu_eu"
+"fiscal_position_hu_eu_out_F27","F27","FEXT","fiscal_position_hu_eu_out"
+"fiscal_position_hu_eu_out_F18","F18","FEXT","fiscal_position_hu_eu_out"
+"fiscal_position_hu_eu_out_F5","F5","FEXT","fiscal_position_hu_eu_out"
+"fiscal_position_hu_eu_out_V27","V27","VIMK","fiscal_position_hu_eu_out"
+"fiscal_position_hu_eu_out_V27TE","V27TE","VIMK","fiscal_position_hu_eu_out"
```
Comment (william-andre) [inline addons/l10n_hu/data/account.fiscal.position.tax.template.csv:21]: Can you configure your IDE to add a EOL at the end of every file (or every syntax used by Odoo)?

File: addons/l10n_hu/data/account_tax_report_data.xml [hunk 2]
```diff
@@ -108,6 +115,14 @@
         <field name="parent_id" ref="tax_report_alap"/>
     </record>
 
+    <record id="tax_report_alap_viss_koron_kivuli" model="account.tax.report.line">
+        <field name="name">Adóalap - Visszaigényelhető ÁFA körön kívüli</field>
+        <field name="tag_name">Adóalap - Visszaigényelhető ÁFA körön kívüli</field>
+        <field name="report_id" ref="tax_report"/>
+        <field name="sequence" eval="4"/>
```
Comment (william-andre) [inline addons/l10n_hu/data/account_tax_report_data.xml:122]: This sequence seems weird considering the position in the file.

File: addons/l10n_hu/data/account_tax_report_data.xml [hunk 3]
```diff
@@ -186,4 +209,11 @@
         <field name="parent_id" ref="tax_report_fizetndo"/>
     </record>
 
-</odoo>
+    <record id="tax_report_fizetndo_viss_komp" model="account.tax.report.line">
+        <field name="name">Visszaigényelhető kompenzációs felár</field>
+        <field name="tag_name">Visszaigényelhető kompenzációs felár</field>
+        <field name="report_id" ref="tax_report"/>
+        <field name="sequence" eval="6"/>
```
Comment (william-andre) [inline addons/l10n_hu/data/account_tax_report_data.xml:216]: Sequence 7?

File: addons/l10n_hu/data/account_tax_template_data.xml [hunk 4]
```diff
@@ -526,32 +867,124 @@
...
+    <record id="VEUM" model="account.tax.template">
+        <field name="description">EU</field>
         <field name="chart_template_id" ref="hungarian_chart_template"/>
         <field name="type_tax_use">purchase</field>
-        <field name="name">Visszaigényelhető – Import</field>
+        <field name="name">EU Adómentes szolgáltatás</field>
         <field name="amount_type">percent</field>
         <field name="amount">0</field>
-        <field name="sequence">2</field>
+        <field name="sequence">1</field>
```
Comment (william-andre) [inline addons/l10n_hu/data/account_tax_template_data.xml:942]: Again, I don't understand the sequences :thinking:

File: addons/l10n_hu/data/account.financial.report.csv [hunk 5]
```diff
@@ -1,43 +0,0 @@
-"id","display_detail","style_overwrite","account_type_ids/id","account_ids/id","parent_id/id","sign","name","account_report_id/id","sequence","type"
```
[the remark this answers is not in the document]
Comment (rugo-odoo) [reply addons/l10n_hu/data/account.financial.report.csv:1]: I have no clue, didn't do anything

**Comment by rugo-odoo** [comment]
@william-andre revision done

File: addons/l10n_hu/data/account_tax_report_data.xml [hunk 6]
```diff
@@ -1,189 +1,246 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-
```
Comment (william-andre) [inline addons/l10n_hu/data/account_tax_report_data.xml:3]: Please remove the whitespace diff in this file (it's fine for the taxes).
While it is interesting to have it, we should do it in 15.1 where it  is truly implemented. In here, it just makes the diff harder to read and fixes harder to rebase through.

**Review by william-andre — COMMENTED** [review]
Small detail

**Comment by rugo-odoo** [comment]
@william-andre revision done

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**